deploy/k8s/charts/trustification/templates/helpers/_infrastructure.tpl

```diff
@@ -1,3 +1,22 @@
+{{/*
+Evaluate the infrastructure port.
+
+Arguments: (dict)
+  * root - .
+  * module - module object
+*/}}
+{{- define "trustification.application.infrastructure.port"}}
```

Comment:
I think it would be simpler to provide actual values in the (default) values file for `.infrastructure`, and then to a `merge (deepCopy)` of the two `InfrastructureConfig` objects. That pattern can be re-used with other infrastructure settings.